server/athenian/api/models/state/env.py

```diff
@@ -27,6 +29,10 @@
 # ... etc.
 
 
+def _sqlalchemy_url_override() -> Optional[str]:
+    return os.environ.get("ALEMBIC_SDB", None)
```

Comment:
We shouldn't modify the env file. It is trivial to patch the config file with `sed` in the CI as needed.